README.md

```diff
@@ -3,26 +3,27 @@ worldcubeassociation.org
 
-## Setup
-- Install [Vagrant](https://www.vagrantup.com/), which requires
-  [VirtualBox](https://www.virtualbox.org/).
+## Setup a development environment
+### Setup a virtual machine and provision it - using Vagrant
+- Install [Vagrant](https://www.vagrantup.com/), which requires [VirtualBox](https://www.virtualbox.org/).
 - `git clone https://github.com/cubing/worldcubeassociation.org` - Clone this repo! (And navigate into it, `cd worldcubeassociation.org`)
 - `git submodule update --init --recursive` - Initialize submodules.
 - `(cd WcaOnRails; bundle install) && pre-commit install` - Set up git pre-commit hook. Optional, but very useful.
-
-## Run
-- `vagrant up noregs` - Once the VM finishes initializing (which can take some time),
-  the website will be accessible at [http://localhost:2331](http://localhost:2331).
+- `vagrant up noregs` - Start the VM and provision it.
   - Note: Starting up the `noregs` vm is much faster than the `all` vm, because the dependencies required to build the WCA regulations take *ages* to install.
+- The website should now be accessible at [http://localhost:2331](http://localhost:2331).
 
-## Provision New VM
-- Provisioning relies upon SSH agent forwarding, so make sure you've set up SSH
-  keys for GitHub ([howto](https://help.github.com/articles/generating-ssh-keys/)).
-  You also need an SSH key set up for cubing@worldcubeassociation.org in order
-  to rsync secrets.
-- `time ssh -A user@example.com 'sudo wget https://raw.githubusercontent.com/cubing/worldcubeassociation.org/master/scripts/wca-bootstrap.sh -O /tmp/wca-bootstrap.sh && sudo -E bash /tmp/wca-bootstrap.sh staging/production'`
+### Install the website on a machine you've setup yourself
+- Setup a machine (ie Ubuntu Trusty) with SSH access
+- Make sure SSH Agent is running or start it with `eval "$(ssh-agent -s)"`
+- Load your GitHub key into the ssh agent by doing `ssh-add /path/to/key`. If you don't have an SSH key, check the [GitHub howto](https://help.github.com/articles/generating-ssh-keys/)
+- SSH into the machine, with agent-forwarding enabled : `ssh -A you@yourmachine`
+- Provison your machine with `sudo wget https://raw.githubusercontent.com/cubing/worldcubeassociation.org/master/scripts/wca-bootstrap.sh -O /tmp/wca-bootstrap.sh && sudo -E bash /tmp/wca-bootstrap.sh [environment]` `[environment]` must be one of `development`, `development-noregs`, `staging`, `production`. This step will take a very long time (at least an hour)
+  - For the `staging` and `production` environment, you will need an SSH key set up or know the password for cubing@worldcubeassociation.org in order to rsync secrets.
+  - The bootstrapping part may fail while trying to install `texlive-lang-all`, in that case you can run `sudo apt-get install texlive-lang-all=2013.20140215-1` manually and then continue the bootstrapping process with `sudo -E bash /tmp/wca-bootstrap.sh [environment]`.
```

Comment:
This is worrisome. Is something timing out? It would be nice to solve this issue if possible.